addons/core/translations/en-us.yaml

```diff
@@ -38,6 +38,9 @@ titles:
   create-resources: Create Resources
   choose-an-option: Choose an option
   feature-edition: 'Edition: {edition}'
+  unit: Unit
+  year: '{numberOfYears, plural, =1 {# year} other {# years}}'
+  days: '{days, plural, =1 {# day} other {# days}}'
```

Comment:
Should we set 0 to `Do not delete` or will this be used in places where that doesn't make sense?

Reply:
I thought about this, and since this can be used in other places I didn't include that.